packages/build-utils/test/unit.stream-to-buffer.test.ts

```diff
@@ -0,0 +1,28 @@
+import { it, describe, expect } from 'vitest';
+import { streamToBufferChunks } from '../src/fs/stream-to-buffer';
+import { Readable } from 'stream';
+
+describe('streamToBufferChunks', () => {
+  it("returns 1 buffer when the total volume doesn't exceed 100mb", async () => {
+    const tenMbs = 10 * (1024 * 1024);
```

Comment:
How about a `const MB = 1024 * 1024` at the top instead of repeating `(1024 * 1024)` a bunch of times in this file?